src/transformStateWithClones.js

```diff
@@ -7,7 +7,50 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let stateArray = [];
+
+  let stateMod = {};
+
+  let i = 0;
+
+  Object.assign(stateMod, state);
+
+  stateArray.push(stateMod);
+
+  actions.forEach(e => {
```

Comment:
```suggestion
  actions.forEach(action => {
```

why `e`?

Reply:
e for me stands for element and looks fancy😊